quicksort/Python/quicksort.cpp

```diff
@@ -0,0 +1,38 @@
+#include<iostream>
+using namespace std;
+int partition(int arr[],int p,int q){
+int k=p;
+for(int i=p+1;i<=q;i++){
+  if(arr[i]<arr[p]){
+    k++;
+    int temp=arr[k];
+    arr[k]=arr[i];
```

Comment:
Line contains following spacing inconsistencies:
- Spaces used instead of tabs.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp4a3uqdoj/quicksort/Python/quicksort.cpp
+++ b/tmp/tmp4a3uqdoj/quicksort/Python/quicksort.cpp
@@ -6,7 +6,7 @@
   if(arr[i]<arr[p]){
     k++;
     int temp=arr[k];
-    arr[k]=arr[i];
+	arr[k]=arr[i];
     arr[i]=temp;
   }
 }
```